themes/bootstrap3/templates/RecordDriver/SolrOverdrive/hold.phtml

```diff
@@ -1,59 +1,81 @@
 <div class="odaction">
+  <?php if($action == 'getTitle'): ?>
+    <h2><?=$this->transEsc($actionTitleCode); ?>
+        <span class="odbrand pull-left">
+          <img class="media-left" src="https://developerportaldev.blob.core.windows.net/media/Default/images/newLogos/OverDrive_Logo_28x28_rgb.jpg">
```

Comment:
Would it be better to include a local copy of this file rather than linking to it externally, or does OverDrive require this approach? The external links may get blocked by content security policies, plus it makes us dependent on external hosting that might change over time.